### ISSUE TITLE
Modules (Under Development)

### ISSUE THREAD

**Issue by pakalinsky** [builtbybel/control-uwp#8]
Just tried to run Ver 0.20.0 on a brand new Win10 laptop.  The "Microsoft Edge" and "Tweaks" modules came up as "Under Development" with no selections available.  I'm assuming this is expected as the modules were pulled for further development and will be released later?  If someone could confirm that I would be most appreciative and will continue to check for future releases.  Thanks.

**Comment by Belim** [comment]
Yes, without any fuss. This is not a bug but a (future) feature :grinning:
I can't tell when i release individual pages/modules

**Comment by Belim** [comment]
Exactly
